sql/core/src/main/scala/org/apache/spark/sql/jdbc/MsSqlServerDialect.scala

```diff
@@ -86,6 +86,35 @@ private object MsSqlServerDialect extends JdbcDialect {
       case "STDDEV_SAMP" => "STDEV"
       case _ => super.dialectFunctionName(funcName)
     }
+
+    override def build(expr: Expression): String = {
```

Comment:
Please don't override `build` which specify the specification of DS V2 framework.

Reply:
See comment above.